<launch>
    <include file="$(find object3d_detector)/launch/object3d_detector_multiclass_no_rviz.launch"/>
    <include file="$(find udp-to-ros)/launch/udp_server.launch"/>

    <node pkg="tracking_accuracy_test" type="tracking_accuracy_test" name="tracking_accuracy_test" respawn="true" output="screen">
        <param name="objectDetectionsTopic" type="string" value="object3d_detector/detections"/>
        <param name="objectDetectionsCoordSystemYAxisUp" type="bool" value="false"/>
        <param name="rvizBaseFrameId" type="string" value="odom"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find tracking_accuracy_test)/launch/object3d_detector.rviz"/>
</launch>